specification/ai/data-plane/ModelInference/preview/2024-05-01-preview/openapi.yaml

```diff
@@ -1353,7 +1353,7 @@ components:
       name: api-key
     BearerAuth:
       type: http
-      scheme: bearer
+      scheme: Bearer
```

Comment:
I really have no idea why this is here.

Reply:
oh thats my change, need to update typespec-next